extension/js/common/browser/ui.ts

```diff
@@ -416,6 +416,5 @@ export class Ui {
   private static swal = () => Swal.mixin({
     showClass: { popup: 'swal2-noanimation', backdrop: 'swal2-noanimation' },
     hideClass: { popup: '', backdrop: '' },
-    scrollbarPadding: false,
```

Comment:
Here I'm undoing #2593 which wasn't the clear fix for #2592

`scrollbarPadding: false` was kinda workaround which is not needed after https://github.com/sweetalert2/sweetalert2/pull/2006 was released

Reply:
Got it, thanks!